src/Interaction.cc

```diff
@@ -397,7 +397,7 @@ void Interaction::get_NN_xsec( double Ek, double &resii, double &resij )
 // hyperon nucleon scatter                                                              
 
 /* Argument list:                                                                      
-Ek hyperon KE in nucleon rest frame                                                    
+Ek hyperon KE in target rest frame                                                    
```

Comment:
target is always nucleon

Reply:
I assume your xsec is hyperon-nucleon, so it should be in nucleon rest frame. Same as all other xsecs here.

Reply:
Okay, I see now. A shortened version of the code on GitHub misled me. Now I understand.